src/UserManager.ts

```diff
@@ -578,7 +570,7 @@ export class UserManager {
     /**
      * Returns promise to trigger a silent request (via an iframe) to the end session endpoint.
      */
-    public async signoutSilent(args: SignoutSilentArgs = {}): Promise<void> {
+    public async signoutSilent(args: SignoutSilentArgs = { silentRequestTimeoutInSeconds: this.settings.silentRequestTimeoutInSeconds }): Promise<void> {
```

Comment:
That should not needed here `silentRequestTimeoutInSeconds: this.settings.silentRequestTimeoutInSeconds`,
its already done here: `IFrameNavigator`:
```
public async prepare({
        silentRequestTimeoutInSeconds = this._settings.silentRequestTimeoutInSeconds,
```

Reply:
Yup, sorry I didn't revert that change by mistake. done.